src/Microsoft.Azure.SignalR.Common/ClientInvocation/IClientResultsManager.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NET7_0_OR_GREATER
+using System;
+using Microsoft.Azure.SignalR.Protocol;
+using System.Threading.Tasks;
+using System.Threading;
+using System.Buffers;
+using Microsoft.AspNetCore.SignalR.Protocol;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal interface IClientResultsManager
```

Comment:
Maybe starting with a PR showing the interface and how the interface is used is more readable... and I am not sure why we need an interface though.... interface is used when you need to mock the implementation in your UT

Reply:
See #1687